docs/templates.md

```
@@ -103,6 +103,84 @@ $view_registry->getNamespacePaths('no-such-namespace');
+
+Turn the three cases into _Aura\View\Exception\ParentNotFound_ while developing:
+
+```php
+<?php
+$view->setStrictParent(true);
+?>
+```
+
+The message names the template and why the lookup came up empty:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)